rest_framework-stubs/fields.pyi

```diff
@@ -127,30 +127,20 @@ class Field(Generic[_VT, _DT, _RP, _IN]):
     def __deepcopy__(self, memo: Mapping[Any, Any]) -> Field: ...
 
 class BooleanField(
-    Field[
-        bool,
-        str | bool | int,
-        bool,
-        Any,
-    ]
-):
-    TRUE_VALUES: set[str | bool | int]
-    FALSE_VALUES: set[str | bool | int | float]
-    NULL_VALUES: set[str | None]
-
-class NullBooleanField(
     Field[
         bool | None,
         str | bool | int | None,
-        bool,
+        bool | None,
         Any,
     ]
 ):
     TRUE_VALUES: set[str | bool | int]
     FALSE_VALUES: set[str | bool | int | float]
     NULL_VALUES: set[str | None]
 
-class CharField(Field[str, str, str, Any]):
+class NullBooleanField(BooleanField): ...
```

Comment:
It looks like `NullBooleanField` was [deprecated in v3.14 of DRF](https://github.com/encode/django-rest-framework/blob/1db19f4b2d1500894448634852e02f73043382e4/docs/community/3.14-announcement.md#deprecations)

Do you want me to remove it in this PR or make a separate one?

Reply:
Separate PR please.